demo/admin/src/products/ProductForm.tsx

```diff
@@ -331,6 +331,8 @@ export function ProductForm({ id }: FormProps) {
                             multiple
                             maxFileSize={1024 * 1024 * 4} // 4 MB
                             fullWidth
+                            layout="grid"
+                            previewImageWidth={640}
```

Comment:
I don't think setting this value should be necessary, maybe it shouldn't even be an option. 
Each Item as a min-width of 200px, so a little more (~250) multiplied by 2 (hi-dpi) should probably be set as the default for this.

https://github.com/vivid-planet/comet/blob/46f93229999393a28673718cb289a3b69707ba15/packages/admin/admin/src/form/file/FileSelect.tsx#L214

Reply:
It isn't necessary, I'm currently using 320 as default (should this be 640?). However, as allowed image resize widths can be defined in the application, it needs to be possible to override the default value.

Reply:
Ah ok 👍🏼 

I think the maximum size the items can have is 230px, so the api should deliver at least twice that width. 
So 460, or whatever the next largest value is, that the api supports. Probably 640 then 👍🏼

Reply:
Changed: 1015071ad52af29d082e97fb1e90158d76a36206